docs/en/sql-reference/10-sql-commands/00-ddl/15-task/01-ddl-create_task.md

```diff
@@ -0,0 +1,131 @@
+ AS
+ INSERT INTO summary_table SELECT * FROM source_table;
+```
+In this example, a task named my_daily_task is created. It uses the compute_wh warehouse to run a SQL statement that inserts data into summary_table from source_table. The task is scheduled to run daily at 9 AM Pacific Time.
+
+```sql
+CREATE TASK IF NOT EXISTS mytask
+ WAREHOUSE = 'system'
+ SCHEDULE = 2 MINUTE
+ SUSPEND_TASK_AFTER_NUM_FAILURES = 3
+AS
+INSERT INTO compaction_test.test VALUES((1));
+```
+This example creates a task named mytask, if it doesn't already exist. The task is assigned to the system warehouse and is scheduled to run every 2 minutes. It will be suspended if it fails three times consecutively. The task performs an INSERT operation into the compaction_test.test table.
+
+```sql
+CREATE TASK IF NOT EXISTS daily_sales_summary
+ WAREHOUSE = 'analytics'
+ SCHEDULE = 30 SECOND
+FROM sales_data
+GROUP BY sales_date;
+```
+In this example, a task named daily_sales_summary is created with a second-level scheduling. It is scheduled to run  every 30 SECOND. The task uses the 'analytics' warehouse and calculates the daily sales summary by aggregating data from the sales_data table.
+
+
+```sql
+CREATE TASK IF NOT EXISTS process_orders
+ WAREHOUSE = 'etl'
+ AFTER task1, task2
+ASINSERT INTO data_warehouse.orders
+SELECT * FROM staging.orders;
+```
+In this example, a task named process_orders is created, and it is defined to run after the successful completion of task1 and task2. This is useful for creating dependencies in a Directed Acyclic Graph (DAG) of tasks. The task uses the 'etl' warehouse and transfers data from the staging area to the data warehouse.
+
+```sql
+CREATE TASK IF NOT EXISTS hourly_data_cleanup
+ WAREHOUSE = 'maintenance'
+ SCHEDULE = '0 * * * *'
+ WHEN STREAM_STATUS('change_stream') = TRUE
+AS
+DELETE FROM archived_data
+WHERE archived_date < DATEADD(HOUR, -24, CURRENT_TIMESTAMP());
+CREATE TASK mytask
+ WAREHOUSE = 'mywh'
+ SCHEDULE = 30 SECOND
+AS
+ INSERT INTO mytable(ts)
+ VALUES(CURRENT_TIMESTAMP);
+
+ ```
+In this example, a task named hourly_data_cleanup is created. It uses the maintenance warehouse and is scheduled to run every hour. The task deletes data from the archived_data table that is older than 24 hours. The task only runs if the change_stream stream contains change data. 
+
+```sql
+CREATE TASK IF NOT EXISTS mytask
+ WAREHOUSE = 'mywh'
+ SCHEDULE = 30 SECOND
+ ERROR_INTEGRATION = 'myerror'
+AS
+BEGIN
+    BEGIN;
+    INSERT INTO mytable(ts) VALUES(CURRENT_TIMESTAMP);
+    DELETE FROM mytable WHERE ts < DATEADD(MINUTE, -5, CURRENT_TIMESTAMP());
+    COMMIT;
+END;
+```
+
+In this example, a task named mytask is created. It uses the mywh warehouse and is scheduled to run every 30 seconds. The task executes a BEGIN block that contains an INSERT statement and a DELETE statement. The task commits the transaction after both statements are executed. when the task fails, it will trigger the error integration named myerror.
```

Comment:
It would be better to add an example that only uses `BEGIN..END`, but doesn't  use `BEGIN; COMMIT;`, which executes multi statment in a task but not in a transaction. Then users may understand the difference between `BEGIN..END` and  `BEGIN; COMMIT;` better.